docker-compose.yml

```diff
@@ -103,4 +103,5 @@ services:
             - ./conftest.py:/src/conftest.py:ro
             - ./tox.ini:/src/tox.ini:ro
             - ./.ddtox:/src/.tox
-        command: bash
+            - ./scripts:/src/scripts
+        command: /src/scripts/run-tox-scenario
```

Comment:
ah, this probably won't do what we want, since running `scripts/ddtest '^futures_contrib-'` won't use `scripts/run-tox-scenario`

I was thinking more about "entrypoint" instead of "command", but that probably isn't the best idea.

Your call here on if we keep this as-is, or do something a little different, or revert back to just `bash` (we still need the `scripts` volume though)

Reply:
Right, with that you get stuck to running all the tests… which is nice but not really handy.

I guess we can keep bash since it's quite generic. I'll update.